kolibri/logger.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+import logging
+
+from colorlog import ColoredFormatter
+
+# TODO: Logging decision.
+# The main Kolibri logger always logs to console and does not propagate.
```

Comment:
At a minimum, I'd like to be able to specify logging to a file. This inability to do this is a great weakness in KA Lite.